### PR TITLE
Fixes #15162: Fix all bugs found with the tests of class_prefix and old class_prefix 

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/15162

### CODE REVIEW

File: tree/30_generic_methods/_file_check_special.cf [hunk 1]
```diff
@@ -23,7 +23,7 @@
 # @parameter file_name File name
 # @parameter file_type File type to check, can be "block device", "character device", "directory", "FIFO/pipe", "regular file", "socket", "symlink"
 # 
-# @class_prefix file_check_special
+# @class_prefix file_check
```
Comment (amousset) [inline tree/30_generic_methods/_file_check_special.cf]: neither of these if actually right, but it is not a real generic method so that's ok

**Comment by amousset** [comment]
@Fdall needs a rebase

**Comment by Fdall** [comment]
PR rebased

**Comment by Fdall** [comment]
OK, merging this PR

**Comment by Fdall** [comment]
OK, merging this PR

**Pull request merged**